addons/pos_restaurant/static/src/js/floors.js

```diff
@@ -218,6 +218,15 @@ var TableWidget = PosBaseWidget.extend({
                         }
                         self.pos.tables_by_id[table.id] = self.table;
                         self.renderElement();
+                        if (self.selected) {
```

Comment:
I tried putting this in `renderElement` but it resulted to flickering of the existing tables in the screen. Wrapping it in `setTimeout` does not result to flickering of the tables, but the new table flickers.